src/lib/crypto/elgamal.cpp

```diff
@@ -158,7 +158,7 @@ elgamal_validate_key(rng_t *rng, const pgp_eg_key_t *key, bool secret)
         goto done;
     }
 
-    if (elgamal_load_secret_key(&bskey, key) ||
+    if (!elgamal_load_secret_key(&bskey, key) ||
```

Comment:
This is an unrelated bug fix, right? Not a problem, but I think I better file an issue to see if we can get a test added.

Reply:
Added reference to #880 where the test is to be implemented.

Reply:
Yep, was surprised to find this line out. Looks like code was incorrect for a while.